tests/scripts/all.sh

```diff
@@ -3774,6 +3777,9 @@ component_test_psa_crypto_config_reference_cipher_aead () {
 
     msg "ssl-opt: crypto config with non-accelerated cipher and AEAD"
     tests/ssl-opt.sh
+
+    msg "compat.sh: crypto config with non-accelerated cipher and AEAD"
```

Comment:
Ah, actually, pre-existing issue: this should say "full config", as well as all other messages that currently have "crypto config" in those two components. I guess we forgot to change that when we expanded the config (which was initially crypto only indeed).

Could you fix it in this PR?

Reply:
Changed my mind let's merge the PR as is since it's approved.

Reply:
Merges are blocked right now due to an incident on readthedocs (probably a configuration change we need to make), so you might as well update.